CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ option(ENABLE_EXAMPLES "Build Umpire examples" On)
 option(ENABLE_LOGGING "Build Umpire with Logging enabled" On)
 option(ENABLE_SLIC "Build Umpire with SLIC logging" Off)
 option(ENABLE_BACKTRACE "Build Umpire with allocation backtrace enabled" Off)
+option(ENABLE_DLSYMS "Build Umpire with symbol support" Off)
```

Comment:
Do you think that this should/could depend on `ENABLE_BACKTRACE`? I think that it should be on by default when backtrace is, which maintains the current behavior.

Also, can you come up with a better name? Maybe `ENABLE_BACKTRACE_SYMBOLS`?

Reply:
Good idea.  This is done.